src/main/resources/assets/create/lang/pl_pl.json

```diff
@@ -0,0 +1,1157 @@
+	"block.create.weathered_limestone_bricks": "Wyblakłe wapienne cegły",
+	"block.create.weathered_limestone_bricks_stairs": "Wyblakłe wapienne ceglane schody",
+	"block.create.weathered_limestone_bricks_wall": "Wyblakły wapienny ceglany murek",
+	"block.create.weathered_limestone_bricks_slab": "Wyblakła wapienna ceglana płyta",
+	"block.create.weathered_limestone_pillar": "Wyblakły wapienny filar",
+	"block.create.weathered_limestone_layers": "Nawarstwiony wyblakły wapień",
+
+	"block.create.dolomite_pillar": "Dolomitowy filar",
+	"block.create.dolomite": "Dolomit",
+	"block.create.dolomite_stairs": "Dolomitowe schody",
+	"block.create.dolomite_wall": "Dolomitowy murek",
+	"block.create.dolomite_slab": "Dolomitowa płyta",
+	"block.create.dolomite_bricks": "Dolomitowe cegły",
+	"block.create.dolomite_bricks_wall": "Dolomitowy ceglany murek",
+	"block.create.dolomite_bricks_stairs": "Dolomitowy ceglane schody",
+	"block.create.dolomite_bricks_slab": "Dolomitowe ceglana płyta",
+	"block.create.polished_dolomite": "Wypolerowany dolomit",
+	"block.create.dolomite_layers": "Nawarstwiony dolomit",
+
+	"block.create.limesand": "Piasek wapienny",
+	"block.create.limestone": "Wapień",
+	"block.create.limestone_stairs": "Wapienne schody",
+	"block.create.limestone_slab": "Wapienna płyta",
+	"block.create.limestone_wall": "Wapienny murek",
+	"block.create.limestone_bricks": "Wapienne cegły",
+	"block.create.limestone_bricks_stairs": "Wapienne ceglane schody",
+	"block.create.limestone_bricks_slab": "Wapienna ceglana płyta",
+	"block.create.limestone_bricks_wall": "Wapieny ceglany murek",
+	"block.create.polished_limestone": "Wypolerowany wapień",
+	"block.create.polished_limestone_slab": "Wypolerowana wapienna płyta",
+	"block.create.limestone_pillar": "Wapienny filar",
+	"block.create.limestone_layers": "Nawarstwiony wapień",
+	
+	"block.create.natural_scoria": "Naturalna scoria",
+	"block.create.scoria": "Scoria",
+	"block.create.scoria_stairs": "Scoriowe schody",
+	"block.create.scoria_slab": "Scoriowa płyta",
+	"block.create.scoria_wall": "Scoriowy murek",
+	"block.create.scoria_bricks": "Scoriowe cegły",
+	"block.create.polished_scoria": "Wypolerowana scoria",
+	"block.create.polished_scoria_slab": "Wypolerowana scoriowa płyta",
+	"block.create.scoria_pillar": "Scoriowy filar",
+	"block.create.scoria_layers": "Nawarstwiona scoria",
+
+	"block.create.dark_scoria": "Ciemna scoria",
+	"block.create.polished_dark_scoria": "Wypolerowana ciemna scoria",
+	"block.create.dark_scoria_tiles": "Ciemne scoriowe kafle",
+	"block.create.dark_scoria_tiles_stairs": "Ciemna scoriowa kaflowa płyta",
+	"block.create.dark_scoria_tiles_slab": "Ciemna scoriowa kaflowa płyta",
+	"block.create.dark_scoria_bricks": "Ciemne scoriowe cegły",
+	"block.create.dark_scoria_bricks_stairs": "Ciemne scoriowe ceglane schody",
+	"block.create.dark_scoria_bricks_slab": "Ciemna scoriowa ceglana płyta",
+	"block.create.dark_scoria_bricks_wall": "Ciemny scoriowy ceglany murek",
+	
```

Comment:
And that is all for now, don't hesitate to @ me if you need a hand with translating something, or you want me to continue the review!

Good job so far!

Reply:
@LTenshi I will use your help as Polish translation reviewer in the future to be sure.